goreleaser.hcl

```diff
@@ -1,29 +1,28 @@
 description = "go release builder"
-test = "goreleaser --version"
-binaries = ["goreleaser"]
+test        = "goreleaser --version"
+binaries    = ["goreleaser"]
 
-version "0.159.0" "0.182.1" {
```

Comment:
I wasn't sure why `0.159.0` is in the block with `auto-version` 🤔

Reply:
Not sure, but you can't delete a version. It'll have to be moved below.

I think maybe than all this, just the auto-version block and 0.182.1 should be moved down to the bottom block where 0.821.0 and 0.182.1 are?